src/community/design-system-day/session-information.md

```diff
@@ -0,0 +1,221 @@
+
+### How brand design systems offer the power of confidence
+Abigail and Chloe Baldwin
+10:00 to 11:30
+
+Amid challenges, two young twin designers from humble backgrounds share a relatable tale of creative business beginnings. Design systems alleviate the burden of risk, functioning as a guide, and empowering the creative practice. This talk will provide practical guidance to fledgling designers and business owners to forge their branding systems.
+
+### How GOV.UK Pay is making recurring payments easy for the public sector
+Sacha Zarb
+10:00 to 11:30
+
+The talk will look at how GOV.UK Pay, has brought functionality to the platform, that allows services to quickly and easily add recurring payments. We will talk about the design challenges, how services are using this, and how we've designed this for the future. 
+
+### From pixels to paradigm shifts – tailoring a dedicated design system to catalyse innovation in the Italian Ministry of Economy and Finance
+Simone di Fresco
+10:00 to 11:30
+
+Unlock the power of dedicated Design Systems with our case study on the Italian Ministry of Economy and Finance. Discover how user insights and innovative tools are supporting the accounting operations of hundreds of thousands of users in the major Italian Public administrations, fostering a culture of meaningful change.
+
+### The challenges of scaling up a truly user-centred service
+Laura Smith
+10:00 to 11:30
+
+Using two case studies from public healthcare, we'll consider the challenges of maintaining a focus on users when scaling beyond minimum viable product (MVP) into growth and diversification.
+
+### A service designer's approach to human-centered life transitions
+Sylvie Abookire 
+10:00 to 11:30
+
+How might we… design our design practice for optimal success and wellbeing? The methods we use every day as designers can help us creatively navigate our own professional journeys. Through reflective, hands-on activities, participants will learn how they can apply human-centered design to navigate personal decisions surrounding life and work.
+
+### Leading the system
+Daniel Fontaneda
+13:45 to 14:45
+
+In this talk I will explain what are the key pillars that are helping me and my team to lead the design system in a complex environment, working with multiple products at the same time and being a small team and dealing with many different stakeholders.
+
+### Imposter syndrome in government and why it's not always a bad thing
+Ciandelle Hughes
+13:45 to 14:45
+
+Imposter syndrome is something that is often talked about but rarely understood well. This session is designed to talk about the basics of imposter syndrome, including the way in which it can manifest itself. 
+
+We will explore some of the ways imposter syndrome can impact your personal and professional lives. This will be both on a short and longer term basis. We will ask ourselves two key questions:
```

Comment:
```suggestion
We will explore some of the ways imposter syndrome can impact your personal and professional lives. This will be both on a short and longer term basis. We will ask ourselves 2 key questions:
```